.github/workflows/checks.yml

```diff
@@ -26,3 +26,15 @@ jobs:
         with:
           cache: yarn
       - uses: ./.github/actions/security/lockfile
+
+  audit-deps:
```

Comment:
Would it be possible to limit the run of that check to changes to the package.json files and yarn lockfile?

Reply:
This could be an option – we could use https://github.com/dorny/paths-filter to then supply a conditional to the `audit-deps.if` statement

Reply:
Cool 💪🏼 One other thing I am wondering: is it sufficient to test this on the monorepo, or would it be better to also create a test-app (similar to e2e) and test on this app?

Reply:
Guess that depends on whether the monorepo fails now on `main` when you try to use `pnpm` 🤔  – i'll check tomorrow. Thanks for sharing the thoughts!!

Reply:
Yep, you can use `yarn3` or `pnpm` in the monorepo. I suppose we'd have to do it with a test app or something... One for another day 😓